db.go

```diff
@@ -131,6 +136,34 @@ func (rr *redisRepo) fetchInstanceEvents(instanceID string) ([]*lifecycleEvent,
 	return events, nil
 }
 
+func (rr *redisRepo) fetchAllInstanceEvents() (map[string][]*lifecycleEvent, error) {
+	conn := rr.cg.Get()
+	defer rr.closeConn(conn)
+
+	instanceEventKeys, err := redis.Strings(conn.Do("KEYS", fmt.Sprintf("%s:instance:*:events", RedisNamespace)))
```

Comment:
For the record, all of the `[]uint8` reported by `%T` were really `[]byte`, with the full results being, e.g.:

``` go
rawResult := []interface{}{
    []byte("3"),
    []interface{}{
        []byte("cyclist:flooh:flah:9000"),
        []byte("cyclist:fleep:flurb:9001"),
    }
}
```